src/components/RulesPage/rules.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable max-len */
+import * as Constants from "@niaefeup/nijobs-commons";
+
+export const rules = Object.freeze([
+    {
+        "title": "Using NIJobs (as a company)",
+        "description": "As a company, you must first apply for an account.The application will be reviewed and, if accepted, you will have an account registered in NIJobs. After that, you can finish you company registration to be able to start posting job offers. Please remember that your offers should be aimed at college students (still graduating or recently graduated).",
```

Comment:
Add space after `account.`

Reply:
Change to "After that, you need to finish your company registration to be" 
Also typo in "you" instead of "your"

Reply:
I would place "offers should be aimed at college students" in offer rules

Reply:
Maybe something like "In order to avoid spam, and to ensure the greatest quality of offers to the students, a company cannot have more than 5 active offers at the same time."